### PR TITLE
Bug 1989876: Fix routing for ODF 4.9(OCS) Dashboard and link Ceph Block Pools

### PR DESCRIPTION
**Pull request by bipuladh**
Screenshot from 2021-08-04 14-48-38
/assign @GowthamShanmugam

### CODE REVIEW

**Comment by SanjalKatiyar** [comment]
https://github.com/openshift/console/blob/6688742107f4adbf1387b933bf7a75f19723fc63/frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-system-dashboard.tsx#L101
this details card should be imported from `/persistent-external/details-card` but here we are using the one imported from `/persistent-internal/details-card`

**Comment by SanjalKatiyar** [comment]
> https://github.com/openshift/console/blob/6688742107f4adbf1387b933bf7a75f19723fc63/frontend/packages/ceph-storage-plugin/src/components/dashboards/ocs-system-dashboard.tsx#L101
> 
> 
> this DetailsCard should be imported from `/persistent-external/details-card` but here we are using the one imported from `/persistent-internal/details-card`

Never mind, we already have a BZ for it now. I will fix this and send a small patch.

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  const pages: Page[] = [
+    {
+      path: 'overview/:dashboard',
+      href: 'overview/block-file',
+      name: t('ceph-storage-plugin~Overview'),
+      component: OCSOverview,
+    },
+    {
+      href: 'pools',
+      name: t('ceph-storage-plugin~Storage pools'),
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx:35]: As discussed with @yuvalgalanti already better keep heading as BlockPools:

 {
      href: referenceForModel(CephBlockPoolModel),
      name: t('ceph-storage-plugin~BlockPools'),

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+      component: () => <BlockPoolListPage namespace={CEPH_STORAGE_NAMESPACE} />,
+    },
+  ];
+
+  const breadcrumbs = [
+    {
+      name: t('ceph-storage-plugin~Storage systems'),
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx:42]:
```suggestion
      name: t('ceph-storage-plugin~StorageSystems'),
```
Its a CRD name

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+      path: '/odf/systems',
+    },
+    {
+      name: t('ceph-storage-plugin~Storage system details'),
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx:46]:
```suggestion
      name: t('ceph-storage-plugin~StorageSystem details'),
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+      path: '',
+    },
+  ];
+
+  const title = (rest.match.params as any).systemName;
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx:51]:
```suggestion
  const title = systemName;
```

File: frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+      href: referenceForModel(CephBlockPoolModel),
+      name: t('ceph-storage-plugin~BlockPools'),
+      component: () => <BlockPoolListPage namespace={CEPH_STORAGE_NAMESPACE} />,
+    },
+  ];
+
+  const breadcrumbs = [
+    {
+      name: t('ceph-storage-plugin~StorageSystems'),
+      path: '/odf/systems',
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/src/components/dashboards/odf-system-dashboard.tsx:45]: is StorageSystem list page is linked to this URL?

File: frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json [hunk 6]
```diff
@@ -379,6 +379,9 @@
   "Storage Efficiency": "Storage Efficiency",
   "OpenShift Container Storage Overview": "OpenShift Container Storage Overview",
   "Block and File": "Block and File",
+  "Storage pools": "Storage pools",
+  "Storage systems": "Storage systems",
+  "Storage system details": "Storage system details",
```
Comment (GowthamShanmugam) [inline frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json:384]: need to run i18n

**Pull request merged**